src/main/features/certificateOfService/routes/file-upload.ts

```diff
@@ -33,6 +35,7 @@ export default express.Router()
 
       form.parse(req)
       .on('file', function (name, file) {
+        const buffer = readChunk.sync(file.path, 0, 4100)
```

Comment:
Since this is a blocking operation it would be nice to wrap it in a promise. I think we could go further than just reading the file and make the whole file type checking a promised operation. Something like `isOfAcceptedMimeType(file): Promise<boolean>`. What do you think?

Reply:
sounds like a good idea. once I get it working with the MS file types I'll refactor it